app.js

```diff
@@ -387,35 +387,43 @@ function sendNotificationToUser(user, message, icon, severity) {
             }
         }
         // If we found any android devices, send notification
-        if (androidRegistrations.length > 0) {
-            sendAndroidNotifications(androidRegistrations, message);
+        if (config.gcm && androidRegistrations.length > 0) {
+            sendAndroidNotifications(androidRegistrations, newNotification);
         }
         // If we found any ios devices, send notification
-        if (iosDeviceTokens.length > 0) {
-            sendIosNotifications(iosDeviceTokens, message);
+        if (config.apn && iosDeviceTokens.length > 0) {
```

Comment:
Would be nice if you would add a new method `isAppleNotificationConfigured` to the system module instead of accessing the config object directly here.

Reply:
I can do both, but please do note I only pulled those checks out of the called methods (to avoid the admittedly minor overhead).

Reply:
Either way: fixed now.